sql/projects_fisa.sql

```diff
@@ -50,8 +50,16 @@ CREATE TABLE cpdb_projects AS(
                c.nccother,
                c.noncitycost,
                c.totalcost,
-               a.cape_agencyacronym as magencyacro,
-               a.cape_agency as magencyname
+               (CASE
+               WHEN upper(s.description) LIKE '%BNY%' AND s.magency = '801' THEN 'BNY'
+               WHEN lower(s.description) LIKE '%governor%s island%' AND s.magency = '801' THEN 'TGI'
+               ELSE a.cape_agencyacronym
+               END) as magencyacro,
+               (CASE
+               WHEN upper(s.description) LIKE '%BNY%' AND s.magency = '801' THEN 'Brooklyn Navy Yard'
+               WHEN lower(s.description) LIKE '%governor%s island%' AND s.magency = '801' THEN 'Trust for Governors Island'
+               ELSE a.cape_agency 
+               END) as magencyname
```

Comment:
This looks good, running SQL queries like 
```
select * from cpdb_projects where magencyname = 'Trust for Governors Island';
select * from cpdb_projects where magencyacro= 'BNY';
```
shows that the governors island/brooklyn navy yard agency is correctly derived from the description.